src/features/components/templates/banners/blade-shape-banner/index.tsx

```diff
@@ -79,31 +79,63 @@ const BladeShapeBanner = ({ data }: { data: BannerType }) => {
                             )}
                         </FlexBox.Box>
                     </FlexBox.Box>
-                </Flex.Box>
-                <Flex.Box
-                    direction={'col'}
-                    align={'center'}
-                    basis="full"
-                    padding_block="10x"
-                    md={{ basis: '5-12', align: 'end' }}
-                >
-                    <div className={styles.download_col}>
-                        <FlexBox.Box
-                            className={styles.qr_box}
-                            align={'center'}
-                            padding={'12x'}
-                            mb={'12x'}
-                            mt={'20x'}
-                            gap="8x"
-                            md={{ mt: '0x' }}
-                        >
+                </FlexBox.Box>
+                <FlexBox.Box direction={'col'} align={'center'}>
+                    {data?.traders_hub_info && (
+                        <Link url={data?.traders_hub_info.url} no_hover>
```

Comment:
hi man i saw 
traders_hub_info: QRScanBoxType  traders_hub_info is a mandatory field do we need ? here

Reply:
it's not about type, it's about how react works @mayuran-deriv 
It's just a good practice to prevent strange bugs

Reply:
hi @NikitK-deriv  but here ?  not needed since your passing data via data folder its static so no need to pass ?